evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -170,3 +166,21 @@ def _handle_partial_dependence_fast_mode(self, pipeline_parameters, X, target):
             # which would happen with the full set of features for a single column at refit
             pipeline_parameters["DFS Transformer"]["features"] = []
         return pipeline_parameters
+
+    @staticmethod
+    def contains_pre_existing_features(
+        dfs_features: List[FeatureBase],
+        input_feature_names: List[str],
+        target: Optional[str] = None,
+    ):
+        if dfs_features:
+            dfs_feature_names = [
+                name
+                for feature in dfs_features
+                for name in feature.get_feature_names()
+                if name != target
+            ]
+            return dfs_feature_names == input_feature_names
+
+        else:
+            return False
```

Comment:
Seems this line isn't being hit by tests - might be worth a simple test in `test_featuretools` to get this covered

Reply:
Done